bin/run_gosec

```diff
@@ -1,14 +1,11 @@
 #!/usr/bin/env bash
 
-set -eo pipefail
```

Comment:
At best, this set of flags should be removed for the exact place where errors are expected and not for the whole pipeline step otherwise a bug in any part of this file will result in the whole security checking passing in Jenkins

Reply:
So the current call pattern is
Jenkinsfile calls check_golang_security
check_golang_security builds a docker container and runs run_gosec
run_gosec runs gosec

I only removed `set -eo pipefail` from run_gosec, it still exists in check_golang_security

Reply:
Currently, it looks like the issue is that pipefail is getting an exit code from gosec that is non zero, but isn't a failure, so gosec is ended abruptly. The only way I can get it to finish scanning master is to remove pipefail from the file that only runs gosec.

Reply:
Currently, it looks like the issue is that pipefail is getting an exit code from gosec that is non zero, but isn't a failure, so gosec is ended abruptly. The only way I can get it to finish scanning master is to remove pipefail from the file that only runs gosec.